logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/ProtoContentMapper.kt

```diff
@@ -376,6 +373,42 @@ class ProtoContentMapperImpl(
         )
     }
 
+    private fun unpackEphemeral(
+        protoContent: GenericMessage.Content.Ephemeral
+    ): MessageContent.FromProto {
+        val messageContent = when (val ephemeralContent = protoContent.value.content) {
+            is Ephemeral.Content.Text -> {
+                val genericMessageTextContent = GenericMessage.Content.Text(
+                    Text(
+                        content = ephemeralContent.value.content,
+                        mentions = ephemeralContent.value.mentions,
+                        quote = ephemeralContent.value.quote,
+                        expectsReadConfirmation = ephemeralContent.value.expectsReadConfirmation
+                    )
+                )
+                unpackText(genericMessageTextContent)
+            }
+
+            is Ephemeral.Content.Asset -> {
+                val genericAssetContent = GenericMessage.Content.Asset(
+                    Asset(
+                        original = ephemeralContent.value.original,
+                        status = ephemeralContent.value.status,
+                        expectsReadConfirmation = ephemeralContent.value.expectsReadConfirmation
+                    )
+                )
+                unpackAsset(genericAssetContent)
+            }
+
+            // TODO: Mateusz : for now we ignore anything else then Text
```

Comment:
Not up-to-date anymore as `Asset` is handled.

AFAIK, the only possible self-deleting messages are:
- Text
- Asset
- Location

We do not support `Location` yet. So we can just remove this TODO. Or make it like:

```suggestion
            // TODO: Handle self-deleting Location messages when they are implemented
```

Reply:
done